modules/natgw/.header.md

```diff
@@ -31,8 +31,8 @@ module "natgw" {
   name                = "NATGW_name"
   resource_group_name = "resource_group_name"
   location            = "region_name"
-  subnet_ids          = { "a_subnet_name" =
-module.vnet.subnet_ids["a_subnet_name"] }
+  subnet_ids          = { "a_subnet_name" = module.vnet.subnet_ids["a_subnet_name"] }
```

Comment:
Here I'd replace the reference to `vnet` module with string-type placeholder too, like:
`{ "subnet_name" = "/subscription/xxxx/......." }`